### ISSUE TITLE
Missing SetDelay() documentation

### ISSUE THREAD

**Issue by PanzerFowst** [arduino-libraries/ArduinoRS485#45]
Not sure if this goes here in the GitHub, but I couldn't figure out how to put it there, but there seems to be at least the `SetDelay()` function documentation missing in https://www.arduino.cc/reference/en/libraries/arduinors485/.  Like, should it be called before a `begin()` call or after or does it not matter?

Thanks!

**Comment by JojoS62** [comment]
I'm just analyzing the RS485 code. The postdelay seems to be necessary for driving the DE signal (enable transmitter in driver) because the _serial->write() returns before all bits are on the line. So the setting is dependent on the bitrate of the uart.

**Comment by PanzerFowst** [comment]
Yeah, that sounds consistent with my experience as well.  I am setting the delay using a simple equation for Modbus:
```cpp
    constexpr auto ModBaud{9600};
    constexpr auto bitduration{1.0f / ModBaud};
    constexpr auto charlen{11.0f};
    constexpr auto delay_us{bitduration * charlen * 3.5f * 1e6};
```

But it would be nice if the default values for delay could be calculated based on the baud of the `RS485`.

**Comment by per1234** [comment]
Hi all. It seems this is going off topic. While it is very interesting, I will request that all further discussion on this issue thread be restricted exclusively to the subject of the missing documentation of the `setDelays` function.

You are welcome to submit separate issues for reports of bugs or feature requests.

General discussion is always welcome over on Arduino Forum:

https://forum.arduino.cc/

**Comment by JojoS62** [comment]
The postdelay is essential for the RS485 to work correctly. When the delay is too short, the driver is turning off DE too early and the message is corrupted. When is delay is too long, it blocks the receiving of the ACK. The delay is performed by a wait_us() which burns cpu cycles and blocks the system. The calculation of the delay time based on the baud rate is important, only the factor 3.5 is not necessary and makes it too long, one wordlength is enough.

I don't understand why this can be off topic, This library is used in Portenta Machine Control, which claims to be a professional product. Currently, I'm evaluating the PMC for usage in industrial environment. And there it needs reliable software and clear documentation. Answers like these do not help and will shorten my evaluation period.
